functions.php

```diff
@@ -408,6 +418,102 @@ public function ajax_comments() {
 
 		exit();
 	}
+
+	/**
+	 * Retrieve the requested `drive_story` posts for The Evidence page.
+	 */
+	public function ajax_evidence_stories() {
+		check_ajax_referer( 'fetch_articles', 'nonce' );
+
+		$categories = array_values( get_terms( array(
+			'taxonomy' => 'wsuwp_university_category',
+			'hierarchical' => false,
+			'fields' => 'id=>slug',
+		) ) );
+
+		// Initial scholarships query arguments.
+		$stories_query_args = array(
+			'post_type' => 'drive_story',
+		);
+
+		if ( $_POST['page'] && is_numeric( $_POST['page'] ) ) {
+			$stories_query_args['offset'] = ( $_POST['page'] * 10 );
+		}
+
+		if ( $_POST['category'] && in_array( $_POST['category'], $categories, true ) ) {
+			$stories_query_args['tax_query'][] = array(
+				'taxonomy' => 'wsuwp_university_category',
+				'field' => 'slug',
+				'terms' => $_POST['category'],
+			);
+		}
+
+		$stories_query = new WP_Query( $stories_query_args );
+
+		if ( $stories_query->have_posts() ) {
+			$stories = array();
```

Comment:
I would move this outside of the `if` conditional so that in the super crazy scenario that there are no posts, the `wp_json_encode()` at the end of this method has an empty array to work with.